ansible/playbooks/cluster_sbd_prep.yaml

```diff
@@ -4,8 +4,12 @@
   become: true
   become_user: root
   pre_tasks:
-    - name: include iscsi storage profile
-      ansible.builtin.include_vars: ./vars/iscsi-storage-profile.yaml
+    - name: Detect cloud platform
+      ansible.builtin.include_tasks:
+        ./tasks/detect-cloud-platform.yaml
+    - name: Detection result
+      ansible.builtin.debug:
+        msg: "Cloud platform appears to be {{ cloud_platform_name }}"
```

Comment:
is it done to have a better debug/log message? Is it more?

Reply:
For easier debugging.  Until this new task list is bedded in I decided to keep this on-screen so we could check that it was detecting the correct cloud platform.